src/Microsoft.OData.Client/DataServiceQueryOfT.cs

```diff
@@ -312,14 +312,41 @@ public DataServiceQuery<TElement> Expand<TTarget>(Expression<Func<TElement, TTar
 
         /// <summary>Requests that the count of all entities in the entity set be returned inline with the query results.</summary>
         /// <returns>A new <see cref="T:Microsoft.OData.Client.DataServiceQuery`1" /> object that has the inline count option set.</returns>
+        [Obsolete("Please use IncludeCount()")]
         public DataServiceQuery<TElement> IncludeTotalCount()
         {
-            MethodInfo mi = typeof(DataServiceQuery<TElement>).GetMethod("IncludeTotalCount");
+            return this.IncludeCount(true);
+        }
+
+        /// <summary>Requests that the count of all entities in the entity set be returned inline with the query results.</summary>
+        /// <returns>A new <see cref="T:Microsoft.OData.Client.DataServiceQuery`1" /> object that has the inline count option set.</returns>
+        /// <param name="countQuery">Whether to include total count.</param>
+        [Obsolete("Please use includeCount(bool countQuery)")]

```

Comment:
use the right case of the method name in the Obsolete attribute message
```diff
- Please use includeCount(bool countQuery)
+ Please use IncludeCount(bool countQuery)
```

Reply:
fixed